pymatgen/io/vasp/inputs.py

```diff
@@ -545,7 +547,8 @@ def get_str(self, direct: bool = True, vasp4_compatible: bool = False, significa
                 lines.append("Lattice velocities and vectors")
                 lines.append("  1")
                 for v in self.lattice_velocities:
-                    lines.append(" ".join(format_str.format(i) for i in v))
+                    # vasp is strict about the format when reading this quantity
+                    lines.append(" ".join(f" {i: .7E}" for i in v))
```

Comment:
If VASP is strict about this, could you add a test that checks for the right float precision?